DMX.Core.Api.Infrastructure.Provision/Brokers/Clouds/CloudBroker.cs

```diff
@@ -12,6 +12,7 @@ namespace DMX.Core.Api.Infrastructure.Provision.Brokers.Clouds
 {
     public partial class CloudBroker : ICloudBroker
     {
+        private const string ENVIRONMENT = "Production";
```

Comment:
Change this to Environment

Reply:
Unfortunately it conflicts with `System.Environment` :( How about `AzureEnvironment`?

Reply:
_Edit:_ also a naming conflict :) using `DmxEnvironment`